src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp

```diff
@@ -137,7 +137,7 @@
   * @param msg BSMMessage that is received
   * @routeable_message not used
   */
- void ODELoggerPlugin::HandleRealTimePublish(BsmMessage &msg,
+ void ODEForwardPlugin::HandleRealTimePublish(BsmMessage &msg,
```

Comment:
@jwillmartin I know you did not change the code at line 145 below with the "new char" allocation but this seems like a fairly serious memory leak (as noted by Sonar) so do you think we can fix it with this PR?  You could do one of:

- Use a unique_ptr
- use stringstream
- Free it up at the end